### DISCUSSION TITLE
Option to change the margin bottom of my parapgraph inside the text editor

### DISCUSSION THREAD

**Discussion by hein-obox** [elementor/elementor#25431] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Sometimes I use a text editor to add content in a small footer at the bottom of the page. It is annoying that the paragraph has a margin bottom, but I don't seem to an option to change the value of the margin-bottom for the paragraph.

### Describe the solution you'd like.

I would like to see a control inside the text editor to set the margin-bottom for both the paragraphs (general) and for the last paragraph.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by rami-elementor** [comment]
Implemented:

![image](https://github.com/user-attachments/assets/9691b9e3-f566-4da3-aa75-7595778cab48)

**Comment by hein-obox** [comment]
Thank you
